pkgs/servers/monitoring/plugins/esxi.nix

```diff
@@ -27,7 +27,7 @@ in python2Packages.buildPythonApplication rec {
     runHook postInstall
   '';
 
-  propagatedBuildInputs = with python2Packages; [ pywbem ];
+  propagatedBuildInputs = with python3Packages; [ pywbem requests ];
```

Comment:
```suggestion
  propagatedBuildInputs = with python3Packages; [ pywbem requests setuptools ];
```

```
[nix-shell:~/.cache/nixpkgs-review/pr-102282]$ ./results/check-esxi-hardware/bin/check_esxi_hardware.py --help
Traceback (most recent call last):
  File "/nix/store/fx2jz12qvfvv3q4ckwqbzyrfgis67sdz-check-esxi-hardware-20200710/bin/.check_esxi_hardware.py-wrapped", line 290, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

Reply:
Lets take this opportunity to add a simple test for that:

```
  checkPhase = ''
    $out/bin/check_esxi_hardware.py --help
  '';
```

Reply:
`setuptools` is a default build dependency, so this test will always succeed during a build, regardless of whether `setuptools` is available during runtime outside a build. Note this is also a bit of an issue with `pythonImportsCheck` and all tests.

Reply:
you might be able to do something like:
```
  checkPhase = ''
    unset PYTHONPATH
    $out/bin/check_esxi_hardware.py --help
  '';
```